src/app/fdctl/run/tiles/fd_pack.c

```diff
@@ -148,13 +151,21 @@ scratch_align( void ) {
 
 FD_FN_PURE static inline ulong
 scratch_footprint( fd_topo_tile_t const * tile ) {
-  (void)tile;
+  fd_pack_limits_t limits[1] = {{
+    .max_cost_per_block        = tile->pack.larger_max_cost_per_block ? LARGER_MAX_COST_PER_BLOCK : FD_PACK_MAX_COST_PER_BLOCK,
+    .max_vote_cost_per_block   = FD_PACK_MAX_VOTE_COST_PER_BLOCK,
```

Comment:
Need to update the solana side to leave this the same as well then

Reply:
I had made the change but not pushed everything. Thanks for catching it!